modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -238,7 +243,7 @@ The `hosts` parameter is a list of separate bare metal assets used to build the
 
 | [[bootMACAddress]]`bootMACAddress`
 |
-| The MAC address of the NIC the host will use to boot on the `provisioning`  network.
+| The MAC address of the NIC the host will use to boot on the `provisioning`  network. The installer will retrieve the IP address using the `bootMACAddress` configuration setting. Ironic binds to the host using the retrieved IP address.
```

Comment:
Try to avoid future tense, maybe "The installer retrieves..."

Reply:
Fixed.